astroid/raw_building.py

```diff
@@ -224,13 +250,18 @@ def object_build_datadescriptor(node, member, name):
 
 def object_build_methoddescriptor(node, member, localname):
     """create astroid for a living method descriptor object"""
-    # FIXME get arguments ?
+    # get arguments infos
+    args, posonlyargs, defaults, kwonlyargs = _get_args_info_from_callable(member)
+
     func = build_function(
-        getattr(member, "__name__", None) or localname, doc=member.__doc__
+        name=getattr(member, "__name__", None) or localname,
+        args=args,
+        posonlyargs=posonlyargs,
+        defaults=defaults,
+        doc=member.__doc__,
+        kwonlyargs=kwonlyargs,
     )
-    # set node's arguments to None to notice that we have no information, not
-    # and empty argument list
```

Comment:
To still be inline with this, I should set `None` to args if we get a `ValueError` calling `signature.signature()`.

Reply:
Yes! Let's do that!